src/dotnet/CommandLine/CommandLineApplication.cs

```diff
@@ -56,15 +56,34 @@ public CommandLineApplication(bool throwOnUnexpectedArg = true)
         public bool HandleRemainingArguments { get; set; }
         public string ArgumentSeparatorHelpText { get; set; }
 
-        public CommandLineApplication Command(string name, bool throwOnUnexpectedArg = true)
+        public CommandLineApplication AddCommand(string name, bool throwOnUnexpectedArg = true)
         {
-            return Command(name, _ => { }, throwOnUnexpectedArg);
+            return AddCommand(name, _ => { }, throwOnUnexpectedArg);
         }
 
-        public CommandLineApplication Command(string name, Action<CommandLineApplication> configuration,
+        public CommandLineApplication AddCommand(string name, Action<CommandLineApplication> configuration,
```

Comment:
NIT: When we put one parameter in a new line, we put each in a new line. Like this

```
AddCommand(
    string name,
    Action<CommandLineApplication> configuration,
    bool throwOnUnexpectedArg = true)
```